File: fix-syntax.py
with open('src/api/mock/routes.ts', 'r', encoding='utf-8') as f:
    content = f.read()

content = content.replace('},,', '},')

with open('src/api/mock/routes.ts', 'w', encoding='utf-8') as f:
    f.write(content)

print("Fixed syntax error")

File: fix-syntax.js
const fs = require('fs');
let content = fs.readFileSync('src/api/mock/routes.ts', 'utf-8');
content = content.replace("          }\n));\n          }\n\n          const behavior", "          }\n\n          const behavior");
fs.writeFileSync('src/api/mock/routes.ts', content, 'utf-8');
